### PR TITLE
Chore/branching update requirements

### PR DESCRIPTION
**Pull request by joshenlim**

### CODE REVIEW

File: apps/studio/components/interfaces/Organization/BillingSettings/Subscription/PlanUpdateSidePanel.tsx [hunk 1]
```diff
@@ -54,7 +54,10 @@ const PlanUpdateSidePanel = () => {
 
   const snap = useOrgSettingsPageStateSnapshot()
   const visible = snap.panelKey === 'subscriptionPlan'
-  const onClose = () => snap.setPanelKey(undefined)
+  const onClose = () => {
+    router.push(router.asPath.split('?')[0], undefined, { shallow: true })
```
Comment (alaister) [inline apps/studio/components/interfaces/Organization/BillingSettings/Subscription/PlanUpdateSidePanel.tsx:58]: Is this just to remove the query string from the URL? If so, I think there might be a better way of doing this

File: apps/studio/components/layouts/AppLayout/EnableBranchingButton/BranchingPlanNotice.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+  return (
+    <Alert_Shadcn_ className="rounded-none px-7 py-6 [&>svg]:top-6 [&>svg]:left-6 border-0 border-y">
+      <AlertCircleIcon />
+      <AlertTitle_Shadcn_ className="text-base">
+        Database branching is only available on the Pro plan and above
+      </AlertTitle_Shadcn_>
+      <AlertDescription_Shadcn_>
+        Head over to your organization's billing settings to upgrade your plan in order to enable
+        branching for this project
```
Comment (alaister) [inline apps/studio/components/layouts/AppLayout/EnableBranchingButton/BranchingPlanNotice.tsx:21]: Love your word choice here, but I'm still going to do a Terry:
```suggestion
        Go to your organization's billing settings and upgrade your plan to enable branching for this project
```
wdyt?

**Review by alaister — APPROVED** [review]
LGTM!

File: apps/studio/components/interfaces/Organization/BillingSettings/Subscription/PlanUpdateSidePanel.tsx (hunk 1 above)
Comment (joshenlim) [reply apps/studio/components/interfaces/Organization/BillingSettings/Subscription/PlanUpdateSidePanel.tsx:58]: ^ yeah it is - what do you have in mind?

File: apps/studio/components/layouts/AppLayout/EnableBranchingButton/BranchingPlanNotice.tsx (hunk 2 above)
Comment (joshenlim) [reply apps/studio/components/layouts/AppLayout/EnableBranchingButton/BranchingPlanNotice.tsx:21]: oh yeah thats way better hahaha

File: apps/studio/components/interfaces/Organization/BillingSettings/Subscription/PlanUpdateSidePanel.tsx (hunk 1 above)
Comment (alaister) [reply apps/studio/components/interfaces/Organization/BillingSettings/Subscription/PlanUpdateSidePanel.tsx:58]: Maybe something like:
```js
const router = useRouter()
const { x, ...queryWithoutX } = router.query
router.push(
  {
    pathname: router.pathname,
    query: queryWithoutX,
  },
  undefined,
  { shallow: true }
)
```

Comment (joshenlim) [reply apps/studio/components/interfaces/Organization/BillingSettings/Subscription/PlanUpdateSidePanel.tsx:58]: ^ ahhh makes sense, will give it a go

Comment (joshenlim) [reply apps/studio/components/interfaces/Organization/BillingSettings/Subscription/PlanUpdateSidePanel.tsx]: okay! updated :) and also updated for the panels in project add ons too

**Review by alaister — APPROVED** [review]

**Review by MildTomato — APPROVED** [review]

**Pull request merged**